src/exprestive.coffee

```diff
@@ -80,20 +86,35 @@ class Exprestive
     @routesMethod @getRoutesHelperMethods()
 
 
+  # Save a function to @reversePaths to get a url back from a route name
+  registerReverseRoute: ({routeName, url}) ->
+    return unless routeName?
+    @reversePaths[camelCase routeName] = (id) ->
+      url.replace ':id', id
+
+
   # A helper method for automatically binding restful controllers in a routes file
   # This is passed as "resources" to the routes file function parameter hash
   resourcesHelperMethod: (controllerName) =>
```

Comment:
A better name seems `resourcesDirective` or whatever the name is for the elements of a routing file.